agent/tests/test_ui.py

```
@@ -0,0 +1,240 @@
+        static.mkdir()
+        (static / "index.html").write_text("<html>hi</html>")
+        (static / "app.js").write_text("console.log('hi')")
+        assets = static / "assets"
+        assets.mkdir()
+        (assets / "style.css").write_text("body{}")
+        monkeypatch.setattr(
+            "iconfucius.client.server._STATIC_DIR", static
+        )
+        return static
+
+    def test_root_resolves_to_index(self, static_dir):
```

Ruff (0.15.2)

[warning] Unused method argument: `static_dir`

(ARG002)